src/main/resources/fabric.mod.json

```diff
@@ -27,10 +27,10 @@
   ],
 
   "depends": {
-    "fabricloader": "*",
-    "minecraft": "1.0.0-beta.7.3"
+    "minecraft": "1.0.0-beta.7.3",
+    "api": "*"
```

Comment:
Example mod shouldn't depend on the full API, just have it available.

Reply:
Standard example mod depends on Fabric API